bus-mapping/src/operation.rs

```diff
@@ -224,11 +226,22 @@ impl fmt::Debug for MemoryWordOp {
 
 impl MemoryWordOp {
     /// Create a new instance of a `MemoryOp` from it's components.
-    pub fn new(call_id: usize, address: MemoryAddress, value: Word) -> MemoryWordOp {
+    pub fn new_read(call_id: usize, address: MemoryAddress, value: Word) -> MemoryWordOp {
```

Comment:
suggest to use old `new` method name, as op itself don't reflect r/w info.  when push into buss mapping, will specify `RW::WRITE` or `RW::READ`

Reply:
Now the write op is different because of the previous value. But I can keep the original name yes.

Reply:
Reverted the name.